codecov.yml

```
@@ -6,6 +6,7 @@
 # the coverage.
 #
 codecov:
+  token: fb52015a-b78e-4d89-a6e0-a4829aea2efa
```

I think that we can experiment with this.
I am not sure this will fix the issue...but we can try.

I think that it would help to add some info about how this token can be retrieved.

```suggestion
  # Under normal circumstances, the token is not needed for
  # open source project.
  # But we got spurious failures so we are trying to explicitly
  # set the token here.
  # To review and update the token check
  # SOME_URL
  token: fb52015a-b78e-4d89-a6e0-a4829aea2efa
```